.github/workflows/main.yml

```diff
@@ -84,12 +84,13 @@ jobs:
           clouds-yaml: ${{ secrets.CLOUDS_YAML }}
           bootstrap-constraints: "arch=amd64 cores=2 mem=4G"
           bootstrap-options: "${{ secrets.FOCAL_BOOTSTRAP_OPTIONS }} --model-default datastore=vsanDatastore --model-default primary-network=VLAN_2764 --config caas-image-repo=rocks.canonical.com/cdk/jujusolutions"
+          juju-channel: 2.9/candidate
```

Comment:
Remove needless work-arounds for the following issues:
[LP#1968849](https://launchpad.net/bugs/1968849) and [LP#1969645](https://launchpad.net/bugs/1969645)